qst/src/main/java/io/deephaven/qst/type/BoxedByteType.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.qst.type;
+
+import io.deephaven.annotations.SingletonStyle;
+import org.immutables.value.Value.Immutable;
+
+/**
+ * The {@link Byte} type.
+ */
+@Immutable
+@SingletonStyle
+public abstract class BoxedByteType extends BoxedTypeBase<Byte> {
+
+    public static BoxedByteType of() {
```

Comment:
We called this `instance()` on `PrimitiveType` implementations. Should we standardize?

Reply:
Updated to `of()`.